src/app/components/ATIAnalytics/canonical/sendBeaconOperaMiniScript.ts

```diff
@@ -1,12 +1,14 @@
 import isOperaProxy from '#app/lib/utilities/isOperaProxy';
 
 const sendBeaconOperaMiniScript = (atiPageViewUrlString: string) => `
-if (${isOperaProxy.toString()}()) {
-  var xhr = new XMLHttpRequest();
-  xhr.open("GET", "${atiPageViewUrlString}", true);
-  xhr.withCredentials = true;
-  xhr.send();
-} 
+    if (${isOperaProxy.toString()}() && !Boolean(window.hasOperaMinScriptRun)) {
```

Comment:
Does this new flag need testing in here: https://github.com/bbc/simorgh/blob/81d22a8d09600cb242cb37366d147c295c75194a/src/app/components/ATIAnalytics/canonical/sendBeaconOperaMini.test.ts#L11 ?